### ISSUE TITLE
Unable to Reproduce Results on ImageNet-R

### ISSUE THREAD

**Issue by linlany** [DawnLIU35/FSCIL-ASP#1]
Hi ,

Thank you for sharing your work. I have been trying to reproduce the results mentioned in your paper on the ImageNet-R dataset, but I am encountering some difficulties.

Specifically, I followed the steps outlined in the repository and used the provided code and pre-trained models. However, my results are different from those reported in the paper. Here is the results I obtained:
```
2024-10-30 18:15:08,881 [trainer.py] => Top1 curve: [81.37, 78.62, 77.81, 74.92, 73.33, 72.49, 71.15, 69.99, 69.6, 68.33, 67.17]               
2024-10-30 18:15:08,881 [trainer.py] => Average Accuracy (Top1): 73.16181818181819   (Harmonic Accuracy): 63.85073416203086 (Old Acc): 80.45400
000000001 (New Acc): 52.928   
```

**Comment by linlany** [comment]
I found that it may be caused by randomly dividing different train/test set and other randomness.
